components/slides/page-with-sidebar.ftd

```diff
@@ -0,0 +1,258 @@
+-- import: ui.fifthtry.com/site-data
+-- import: ui.fifthtry.com/user-data
+-- import: ui.fifthtry.com/components/header
+-- import: fastn/processors as pr
+-- import: fastn-community.github.io/svg-icons/phosphor
+
+-- boolean unimplemented: true
+
+-- record toc-item:
+caption title:
+string link:
+
+-- component page:
+module site-data: site-data
+module user-data: user-data
+toc-item list list-of-toc: $stable-toc-links
+boolean $sidebar: true
+string team-name:
+children uis:
+boolean hide-team: false
+
+-- ftd.column:
+colors: $ds.colors
+types: $ds.types
+width: fill-container
+height: fill-container
+background.solid: $inherited.colors.background.base
+
+-- header.header:
+site-data: $page.site-data
+user-data: $page.user-data
+product: slides
+$sidebar: $page.sidebar
+
+-- ftd.row:
+width: fill-container
+background.solid: $inherited.colors.background.base
+
+-- ftd.column:
+if: { page.sidebar }
+border-right-width.px: 1
+border-color: $inherited.colors.border
+sticky: true
+top.px: 84
+left.px: 0
+min-width.fixed.percent: 20
+min-height.fixed.calc: 100vh - 138px
+overflow-y: auto
+width.fixed.percent: 20
+align-content: top-left
+
+-- ftd.row:
+if: { !page.hide-team }
+width: fill-container
+padding-left.px: 24
+padding-right.px: 8
+padding-vertical.px: 16
+border-bottom-width.px: 1
+border-color: $inherited.colors.border
+spacing.fixed.px: 8
+
+-- ds.copy-regular: 
+text: Team:
+align: inline
+
+-- ftd.row:
+spacing.fixed.px: 4
+align-content: center
+
+-- ds.copy-regular: 
+text: $page.team-name
+
+-- ftd.image:
+if: { !unimplemented }
+src: $assets.files.assets.slides.toggle-down.svg
+width.fixed.px: 24
+align-self: center
+
+-- end: ftd.row
+
+-- ftd.column:
+if: { !unimplemented }
+align-self: center
+background.solid: $inherited.colors.background.step-1
+border-radius.px: 8
+border-width.px: 1
+border-color: $inherited.colors.border
+padding.px: 4
+margin-left.px: 32
+color: $inherited.colors.text
+
+-- phosphor.light: gear
+size: 32
+
+-- end: ftd.column
+
+-- end: ftd.row
+
+-- ftd.column:
```

Comment:
Every time you use `ftd.*` you write a comment about why you need it, and why `ds.*` does not have what you need.